README.md

```diff
@@ -1 +1,42 @@
-# manticoresearch-buddy
+# Buddy
+
```

Comment:
```suggestion
Manticore Buddy is a Manticore Search's sidecar written in PHP which helps it with various tasks. The typical workflow is that before returning an error to the user Manticore Search asks Buddy if it can handle it for the daemon. Buddy is written in PHP which makes it easy to implement high-level functionalities that do not require extra-high performance.

```